_config.yml

```diff
@@ -17,7 +17,7 @@ title: ChickTech Favorite Foods
 email: bihan.zh@gmail.com
 description: >- # this means to ignore newlines until "baseurl:"
   What are some of ChickTechRDU's favorite foods?
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: "/ChickTechRDU.github.io"
```

Comment:
oops, don't forget to change this back to the original value! And try to keep the entire comment as it looked before.

Reply:
its fixed